qualtran/bloqs/chemistry/pbc/first_quantization/select_and_prepare.py

```diff
@@ -270,6 +274,10 @@ def selection_registers(self) -> Tuple[Register, ...]:
     def junk_registers(self) -> Tuple[Register, ...]:
         return (Register("succ_nu", QBit()), Register("plus_t", QBit()))
 
+    @property
+    def l1_norm_coeffs(self) -> Optional[SymbolicFloat]:
```

Comment:
no `Optional` here? Perhaps raise a `ValueError` if `self.sum_of_l1_coeffs` is None?

Reply:
Done.

Reply:
Done.